pkg/reconciler/testing/functional.go

```diff
@@ -999,6 +1000,15 @@ func WithPAStatusService(svc string) PodAutoscalerOption {
 	}
 }
 
+func WithMinScale(minScale int) PodAutoscalerOption {
```

Comment:
since it's not used outside single unit test file, probably define it there as a package private.